### PR TITLE
Add paru as an AUR alternative

### PR DESCRIPTION
**Pull request by pblgomez**
As the main yay developer is leaving yay behind and focusing on paru (written in rust) and already hit version 1.0, It is a good alternative to yay.

### CODE REVIEW

**Comment by picodotdev** [comment]
I have see this comment of the main yay developer.

https://aur.archlinux.org/packages/yay/#pinned-788241

He says that yay is still maintained and will keep receiving updates.

I have some questions.

Dou you have a link where do you have readded that main yay developer is leaving yay behind and focusing on paru? paru main developer is Morganamilo, Jguer is developing also paru?
I can add paru as another AUR utility, is there any benefit for set it as default AUR utility in alis?

**Comment by pblgomez** [comment]
You can see the main developer talking about para right here:
https://www.reddit.com/r/archlinux/comments/jjn1c1/paru_v100_and_stepping_away_from_yay/

File: alis-packages.conf [hunk 1]
```diff
@@ -48,7 +48,7 @@ PACKAGES_SDKMAN_SDKS="!gradle !maven !kotlin !groovy" # format <candidate>:[vers
 ## AUR utility and packages to install
 ## more at https://aur.archlinux.org/packages/
 PACKAGES_AUR_INSTALL="false"
-PACKAGES_AUR_COMMAND="yay !aurman"
+PACKAGES_AUR_COMMAND="paru !yay !aurman"
```
[the remark this answers is not in the document]
Comment (pblgomez) [reply alis-packages.conf:51]: Actively mantained, colored output, flip search order, you can edit AUR packages through paru, etc...

File: alis-packages.sh [hunk 2]
```diff
@@ -85,7 +85,7 @@ function check_variables() {
     check_variables_boolean "PACKAGES_FLATPAK_INSTALL" "$PACKAGES_FLATPAK_INSTALL"
     check_variables_boolean "PACKAGES_SDKMAN_INSTALL" "$PACKAGES_SDKMAN_INSTALL"
     check_variables_boolean "PACKAGES_AUR_INSTALL" "$PACKAGES_AUR_INSTALL"
-    check_variables_list "PACKAGES_AUR_COMMAND" "$PACKAGES_AUR_COMMAND" "yay aurman" "true"
+    check_variables_list "PACKAGES_AUR_COMMAND" "$PACKAGES_AUR_COMMAND" "paru !yay aurman" "true"
```
[the remark this answers is not in the document]
Comment (pblgomez) [reply alis-packages.sh]: true

**Pull request closed without merging**